### PR TITLE
add some explanations for conda install and launching the gui

### PR DESCRIPTION
**Pull request by ntorresalberto**
Update the readme with some instructions, as suggested in https://github.com/Gepetto/gepetto-viewer/issues/168.

Also added how to launch the viewer.

### CODE REVIEW

**Comment by hrp2-14** [comment]
Hi ! This project doesn't usually accept pull requests on master. If this wasn't intentionnal, you can change the base branch of this pull request to devel (No need to close it for that). Best, a bot.

**Comment by ntorresalberto** [comment]
Should I change the base to `devel` branch for the PR?

**Comment by nim65s** [comment]
yes please

**Comment by ntorresalberto** [comment]
@jcarpent did it already

**Review by nim65s — APPROVED** [review]

**Review by jcarpent — APPROVED** [review]
Small change requested but it looks very good. Thanks for your help.

File: README.md [hunk 1]
```diff
@@ -4,7 +4,19 @@
 [![Pipeline status](https://gitlab.laas.fr/gepetto/gepetto-viewer/badges/master/pipeline.svg)](https://gitlab.laas.fr/gepetto/gepetto-viewer/commits/master)
 [![Coverage report](https://gitlab.laas.fr/gepetto/gepetto-viewer/badges/master/coverage.svg?job=doc-coverage)](http://projects.laas.fr/gepetto/doc/gepetto/gepetto-viewer/master/coverage/)
 
-## Setup
+## Setup Conda Environment
+
+To try this package in an [conda](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-with-commands) environment without compiling:
+```sh
+conda install gepetto-viewer gepetto-viewer-corba -c conda-forge
```
[the remark this answers is not in the document]
Comment (ntorresalberto) [reply README.md:11]: > gepetto-viewer-corba is sufficient ;)

Indeed, I just realized.
I understand `gepetto-viewer-corba` provides an interface to use it remotely, but I find it a bit confusing that you would need another package to use the viewer.
Is there a way to use `gepetto-viewer` stand-alone? 

If that's the case, maybe a better choice would be to provide 2 separate setup instructions:  **Use Gepetto-viewer remotely from Python** and **Use Gepetto-viewer Standalone**?

Another option is to just add a note clarifying that `gepetto-viewer-corba` is only required when you want to use it remotely like, for instance, with `pinocchio`.

[the remark this answers is not in the document]
Comment (jmirabel) [reply README.md:11]: Theoretically, you could use it from the internal Python console and would not need gepetto-viewer-corba. However, nobody does this so I am not sure it is completely working.

**Pull request merged**